src/drm.cpp

```diff
@@ -1805,6 +1840,16 @@ int drm_prepare( struct drm_t *drm, bool async, const struct FrameInfo_t *frameI
 		if (ret < 0)
 			return ret;
 
+		drm->connector->pending.colorspace = g_bOutputHDREnabled ? DRM_MODE_COLORIMETRY_BT2020_RGB : DRM_MODE_COLORIMETRY_DEFAULT;
```

Comment:
Does `DRM_MODE_COLORIMETRY_BT2020_RGB` work for you? How about `DRM_MODE_COLORIMETRY_BT2020_YCC`? IIRC amdgpu will default to YCC when it's available.

Reply:
If I am honest, I have not seen any output difference with the DP colorimetry packet. Every monitor I have encountered seems to assume BT2020 whenever the PQ EOTF is selected.

It's on my to-do to investigate it further. I'll make a tracking issue.

Reply:
https://github.com/Plagman/gamescope/issues/732